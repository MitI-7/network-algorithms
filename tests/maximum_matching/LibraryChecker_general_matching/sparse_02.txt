500 88 61
344 390
430 179
32 471
316 374
476 393
21 49
376 267
199 15
274 166
276 23
434 474
13 47
10 152
56 363
233 224
109 73
40 21
275 241
432 374
199 451
394 256
332 103
392 423
410 433
375 413
305 367
212 239
138 71
113 237
477 198
207 63
57 19
227 375
147 136
371 489
152 361
427 181
353 351
240 333
154 380
351 90
173 302
397 19
49 456
211 220
202 434
420 63
464 103
452 498
483 413
222 130
310 255
64 132
179 202
240 130
259 154
463 185
115 395
96 470
113 409
43 47
36 274
233 259
382 368
492 451
318 340
378 351
365 439
359 350
425 47
120 243
77 103
77 185
90 319
120 258
158 482
479 184
174 263
367 492
189 127
134 158
87 232
367 108
228 143
359 281
73 457
72 406
135 80