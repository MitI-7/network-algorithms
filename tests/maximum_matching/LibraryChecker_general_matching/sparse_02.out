61
10 152
13 47
15 199
19 57
21 40
23 276
32 471
36 274
49 456
56 363
63 207
64 132
71 138
72 406
73 109
77 103
80 135
87 232
90 319
96 470
108 367
113 237
115 395
120 243
127 189
130 222
134 158
136 147
143 228
154 380
173 302
174 263
179 430
181 427
184 479
185 463
198 477
202 434
211 220
212 239
224 233
227 375
240 333
241 275
255 310
256 394
267 376
281 359
316 374
318 340
344 390
351 353
365 439
368 382
371 489
392 423
393 476
410 433
413 483
451 492
452 498
